Revamps the Science Outpost and slightly modifies lavaland atmos

Isaacnml opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

Revamps the Science Outpost, turning it into the Abandoned Engineering Wing
Giving a lot of space for anyone that wishes to fix it!

It keeps the science shuttle since science/crew usually use it

This PR now also slightly modifies lavaland atmos, giving it 0.25 moles of regenerating plasma
Providing an extra incentive to use the Engineering Wing in lavaland.


## Why It's Good For The Game

The Science Wing was old and there was little to no reason to use it

Let's change that and give engineering or any crew some reason to go down and fix it
The limits are your own imagination 

## Testing Photographs and Procedure
<!-- Include any screenshots/videos/debugging steps of the modified code functioning successfully, ideally including edge cases. -->
<details>
<summary>Screenshots&Videos</summary>

![post1 2](https://github.com/user-attachments/assets/d2caa314-ba60-4209-bafc-ea0b4cafcf83)

![post2 2](https://github.com/user-attachments/assets/8930f990-63ed-4ea1-8d48-1ca67c2a7718)

![post3](https://github.com/user-attachments/assets/904c1faf-351c-4330-8fff-946dc629283d)

![post4](https://github.com/user-attachments/assets/4549887e-fdd4-49da-baa2-c11711020572)




</details>

## Changelog
:cl: Isaac, Cursedkiller


del: Removes the old Science Outpost
adds: A new Engineering related Wing
adds: A new structure near the old Science Outpost, related to mainly farming lavaland gasses
fixes: Lavaland Toxins Test Site Camera facing the wrong way
fixes: Mining Base Injector not working due to it not being connected to the main area
tweak: Lavaland Atmos, providing 0.25 moles of plasma
tweak: Ashwalkers Lungs so they dont die from the small quantities of plasma

/:cl:

Isaacnml committed:
Revamps the Science Outpost Section

Isaacnml committed:
Creates new Areas

Isaacnml committed:
Yikes

Minor fixes

Isaacnml committed:
Ickity Ick

Moves things around, removes extra vent/scrubber in the main room, adds missing decal

Isaacnml committed:
I hate failling checks!!

Isaacnml committed:
Modifies lavaland atmos

Isaacnml committed:
Modifies Ashies lungs

Stops Ashies from just inhaling plasma to death

Isaacnml committed:
Merge branch 'BeeStation:master' into Science-Outpost-Revamped

Isaacnml committed:
I forgor

It shouldn't be 340K

Isaacnml committed:
I need to stop being dumb, they were set to 300!!

I hate failling checks!!

Isaacnml committed:
Still set 340, be 300  c'mon

Isaacnml committed:
Merge branch 'BeeStation:master' into Science-Outpost-Revamped

Isaacnml committed:
I forgot a singular tile

Rukofamicom reviewed (commented):
Aside from key, I'd like assurances that the traces of plasma won't affect ashwalkers or golems in any negative way.

Isaacnml committed:
Adds a small note about plasma for the future.

Isaacnml committed:
Merge branch 'BeeStation:master' into Science-Outpost-Revamped

Isaacnml committed:
Failling Lints

Isaacnml committed:
Lowers the health of the barricades

Isaacnml committed:
I hate failling lints

Isaacnml committed:
Merge branch 'Science-Outpost-Revamped' of https://github.com/Isaacnml/BeeStation-Hornet into Science-Outpost-Revamped

Isaacnml committed:
I frigging hate Lints

BriggsIDP reviewed (commented):
Needs testing for heat, but this area would look much cooler with a catwalk and stuff
![image](https://github.com/user-attachments/assets/573b99f9-fc03-4762-aa2e-20a707c5fac5)

cable
![image](https://github.com/user-attachments/assets/165ad990-c239-4fda-9a4c-07a23a27f9ec)

dead mouse could be nice :)
![image](https://github.com/user-attachments/assets/25bcb440-7814-4c28-8ad9-37d591fc9b7e)

probably want a discharged helper on these apcs
![image](https://github.com/user-attachments/assets/1aad6a2e-5e56-41fb-985b-405fc12bc64e)

could use a few tile_brakers
![image](https://github.com/user-attachments/assets/42fc4d84-ae1c-4219-933e-f03559f912b6)

looks good though!

Isaacnml committed:
Floria's Suggestions and minor additions

Adds a welding tank to a box
Adds foam to a wall, leaving the option to add an airlock
Adds a warning to the reinforced lattices outside, since they dont protect from the lava if you step on them

Isaacnml committed:
Merge branch 'BeeStation:master' into Science-Outpost-Revamped

Isaacnml committed:
Crazy? I'm not Crazy.

Isaacnml committed:
Maybe i am going crazy?

Isaacnml committed:
I beg of thee

Isaacnml committed:
There is no workarounds

Isaacnml committed:
Please please

Isaacnml committed:
My own fault...

Isaacnml committed:
Done goofed

Isaacnml committed:
This should fix the failling lint?
